src/styles/blocks/card.scss

```
@@ -0,0 +1,107 @@
+.card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include hover(
+    (
+      transform: scale(1.2),
+    )
+  );
+
+  &__container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-position: center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 0 0 4px;
```

```suggestion
    margin-bottom: 4px;
```
same in the product-code